transmart-data/ddl/postgres/deapp/de_subject_sample_mapping.sql

```diff
@@ -47,3 +47,15 @@ CREATE INDEX de_subject_smpl_mpng_idx2 ON de_subject_sample_mapping USING btree
 --
 CREATE INDEX idx_de_subj_smpl_trial_ccode ON de_subject_sample_mapping USING btree (trial_name, concept_code);
 
+--
+-- add documentation
+--
+COMMENT ON TABLE deapp.de_subject_sample_mapping IS 'Table links patients to Highdim samples';
+
+COMMENT ON COLUMN de_subject_sample_mapping.patient_id IS 'The patient id linking the patient_dimension';
+COMMENT ON COLUMN de_subject_sample_mapping.subject_id IS 'links to subject_id in de_subject_microarray_data';
```

Comment:
Not correct. Should be:
Corresponds to a part of the sourcesystem_cd column of patient_dimension. The patient_id column should be used for properly referencing patients.